content/integrations/network.md

```diff
@@ -27,6 +27,9 @@ The network check is enabled by default. If you would like to make any changes t
         # matching the given regex:
         # excluded_interface_re: my-network-interface.*
 
+**Note:** To collect connection state metrics, enable the option `collect_connection_state:` by changing it to **true**.
```

Comment:
I've been putting shorter options explanations like this in comments in the YAML, e.g.

`- collect_connection_state: false # set to true to collect TCP connection state metrics, e.g. SYN_SENT, ESTABLISHED